src/time_delta.rs

```diff
@@ -62,6 +62,28 @@ pub struct TimeDelta {
     nanos: i32, // Always 0 <= nanos < NANOS_PER_SEC
 }
 
+#[cfg(feature = "serde")]
+impl serde::Serialize for TimeDelta {
```

Comment:
Nit: I'd prefer to have the `serde` import at the top of the file (with a `cfg` guard if need be), to keep this code a little clear/easier to read.

Reply:
What's noisy here? `serde::Serialize`? Would you prefer just `imply Serialize` here? Also, if we import the namespace of `serde` at the top, we will have one additional `#[cfg(feature = "serde")]`, and this one won't go.
I think, though, the implementations might be under a `mod` which has just one `#[cfg(feature = "serde")]` and all the imports there and all the tests. Would it work?

Reply:
Yes, the `serde::` prefixes throughout these `impl`s. I'm less worried about the `cfg` guards, but agreed that an inline `mod` might be nicer here.

Reply:
Thanks for the clarification.

I will try to do it later today if the original author won't be able to do it by that time. But this will be a separate PR, as I can't push to the author's branch.